CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * [Feature] Added a line under `ResultSet` to distinguish it from data frame and error message when invalid operations are performed (#468)
 * [Feature] Moved `%sqlrender` feature to `%sqlcmd snippets` (#647)
 * [Feature] Added tables listing stored snippets when `%sqlcmd snippets` is called (#648)
+* [Feature] Modified `%load_ext sql` to load saved configurations from `pyproject.toml` file (#689)
```

Comment:
frame your changelog entries so that it's understandable to the end-user, suggestion: 
```suggestion
* [Feature] Allow loading configuration values from a `pyproject.toml` file upon magic initialization (#689)
```

Reply:
Made the change